addons/debugger/addon.json

```diff
@@ -79,13 +83,27 @@
       "id": "fancy_graphs",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Stop previous timing on start timing",
+      "id": "auto_stop_timing",
+      "type": "boolean",
+      "default": false,
+      "description": "Automatically stops the previous timing block when a new one starts—ideal for timing consecutive sections. Only the final block needs an explicit stop."
+    },
+    {
+      "name": "Ratio Time instead of Percentage Time",
+      "id": "show_ratio_time",
+      "type": "boolean",
+      "default": false,
+      "description": "Replaces the percentage column with a ratio of each block’s total time to the smallest block’s total time."
     }
   ],
   "tags": ["editor", "codeEditor", "featured"],
   "versionAdded": "1.16.0",
   "latestUpdate": {
     "version": "1.44.0",
-    "temporaryNotice": "The debugger window is now resizable."
+    "temporaryNotice": "New timing tab to help you with micro-optimisation of your project and the debugger window is now resizable."
```

Comment:
```suggestion
    "temporaryNotice": "New timing tab to help you with micro-optimization of your project and the debugger window is now resizable.",
    "newSettings": ["auto_stop_timing", "show_ratio_time"],
    "isMajor": true
```

Reply:
I thought using US English was in the docs somewhere but I can't find it right now.

Reply:
> I thought using US English was in the docs somewhere but I can't find it right now.

It should probably be added to the docs then. We've always used American spelling, which make sense because Scratch also uses it.

Reply:
I can test it again and approve but I don't have the knowledge to properly review everything here.